### PR TITLE
[Refactor] log message on InstantiatePanelViewModel; fixes #979

### LINKED ISSUE #979: loading times shown in info toolbar are infinite precision
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the COMET-IME
- [x] I have searched [open](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
The user feedback given in the toolbar, on loading times, is shown in infinite precision, which looks weird. Cosmetic issue, and probably simple to fix. Please no more than 3 digits after the dot (msec level). Check for same problem in other views too....

### Steps to Reproduce

1. Open model
2. Go to "Model" view
3. Press "Element Definitions" icon in ribbon
4. Check the Info toolbar at the bottom of the UI: it shows "took xx:xx:xx.yyyyyyyyy" 

### Logs

  - [x] I have attached a log file or included the relevant section in this issue  

image

### System Configuration

- COMET version:
  - [x] COMET-IME: v9.0.1
- Web Services:
  - [x] CDP4/COMET Web Services version: v7.0.2
- Environment (Operating system, version and so on):
  - [x] Windows OS (32/64 bit): W10 (latest patches)

### PR DESCRIPTION
**Pull request by samatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description

  - Refactor the `InstantiatePanelViewModel` logging statement to use `Timespan.ToString("g")`
  - increase code coverage
  - update copyright statements in selected classes

### CODE REVIEW

File: BasicRdl/ViewModels/Ribbons/CategoryRibbonViewModel.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public static IPanelViewModel InstantiatePanelViewModel(ISession session, IThing
             var stopWatch = Stopwatch.StartNew();
             var viewModel = new CategoryBrowserViewModel(session, session.RetrieveSiteDirectory(), thingDialogNavigationService, panelNavigationService, dialogNavigationService, pluginSettingsService);
             stopWatch.Stop();
-            Logger.Info("The CategoryBrowserViewModel opened in {0} [ms]", stopWatch.Elapsed);
+            Logger.Info("The CategoryBrowserViewModel opened in {0}", stopWatch.Elapsed.ToString("g"));
```
Comment (nathanatrhea) [inline BasicRdl/ViewModels/Ribbons/CategoryRibbonViewModel.cs:71]: The [ms] part was removed, I'm not sure it was intentional.

File: CDP4DiagramEditor/ViewModels/DiagramEditorRibbonViewModel.cs [hunk 2]
```diff
@@ -65,7 +68,7 @@ public static DiagramBrowserViewModel InstantiatePanelViewModel(Iteration iterat
             var stopWatch = Stopwatch.StartNew();
             var viewModel = new DiagramBrowserViewModel(iteration, session, thingDialogNavigationService, panelNavigationService, dialogNavigationService, pluginSettingsService);
             stopWatch.Stop();
-            Logger.Info("The DiagramBrowserViewModel opened in {0} [ms]", stopWatch.Elapsed);
+            Logger.Info("The DiagramBrowserViewModel opened in {0}", stopWatch.Elapsed.ToString("g"));
```
Comment (nathanatrhea) [inline CDP4DiagramEditor/ViewModels/DiagramEditorRibbonViewModel.cs:71]: Also here (the [ms]).

File: CDP4Grapher/ViewModels/GrapherRibbonViewModel.cs [hunk 3]
```diff
@@ -34,11 +36,18 @@ namespace CDP4Grapher.ViewModels
 
     using CDP4Dal;
 
+    using NLog;
+
     /// <summary>
```
Comment (nathanatrhea) [inline CDP4Grapher/ViewModels/GrapherRibbonViewModel.cs:37]: I did commented on some missing [ms] strings. But I believe it's the other way around. When using the "g" format specifier on a TimeSpan specifying the unit doesn't make sense anymore.

**Review by nathanatrhea — APPROVED** [review]
I did leave comments on missing "[ms]". But I think it's the other way around. When using the "g" format specifier on a TimeSpan, specifying the unit doesn't make sense anymore to me.

File: BasicRdl/ViewModels/Ribbons/CategoryRibbonViewModel.cs (hunk 1 above)
Comment (lxatrhea) [reply BasicRdl/ViewModels/Ribbons/CategoryRibbonViewModel.cs:71]: Are you also sure you don't want to show the milliseconds part anymore?
```
new TimeSpan(16:14:30)

g                16:14:30
G      0:16:14:30.0000000
```

**Review by lxatrhea — APPROVED** [review]

**Review by nathanatrhea — APPROVED** [review]

**Pull request closed without merging**